container/README.md

```diff
@@ -0,0 +1,53 @@
+# Celix dev container usage
+
+This subdirectory contains a [Containerfile](./Containerfile.gitpod) for developing on Celix locally or via Gitpod.
+Since the image is based on the Gitpod workspace image, it is also usable via https://gitpod.io/#https://github.com/apache/celix
+
+The below steps only need to be executed if you want to develop locally, and not via Gitpod.
+
+## Using a pre-built image
+
+### Start locally with sshd daemon
+
+```bash
+cd <project-root>
+
+# Start a local container with the SSH daemon running
+./container/start-gitpod-container.sh
+```
+
+Now connect to the container via the remote container option of your favoured IDE and start building/developing.
+
+### Start locally with only a bash shell
+
+The start script allows passing of additional paramters, which will override the starting of the SSH daemon.
+Execute the following commands to open a bash shell and build Celix from the command line:
+
+```bash
+cd <project-root>
+
+# Start a local container and open a bash shell
+./container/start-gitpod-container.sh bash
+
+# Build Apache Celix
+mkdir celix-build
+cd celix-build
+../container/support-scripts/build-all.sh
+make -j
+
+# Run the unit tests for Apache Celix
+ctest --output-on-failure
+```
+
+## Build the celix-dev image yourself
+
+```bash
+podman build -t docker.io/apache/celix-dev:$(date +%Y%m%d)-gitpod -f Containerfile.gitpod .
+```
+
+## Deploy a new celix-dev image (Celix PMC members only)
+
+```bash
+podman login docker.io
+podman push docker.io/apache/celix-dev:$(date +%Y%m%d)-gitpod
```

Comment:
I am not sure how easy it is to automate this. I think, since the dependencies don't change that often, that initially it would be okay to push this manually.

Building this every time takes unnecessary build time IMO and also the versioning is not tightly coupled to a Celix release.